bin/curate-trees.py

```diff
@@ -0,0 +1,203 @@
+            "you should read the related FAQ item at "
+            "http://www.iqtree.org/doc/Frequently-Asked-Questions"
+        ),
+    )
+
+    parser.add_argument(
+        "--discardLengths",
+        action="store_true",
+        help=(
+            "When collapsing edges due to low support, the length of the edge being "
+            "removed will be added to the length of the edges leading to the nodes of "
+            "the edge-to-be-removed's child edges. This may be hard to follow, in "
+            "which case you should draw yourself a little picture of part of a rooted "
+            "tree and think about what happens to the nodes that become detached when "
+            "you remove an edge (and the node that it leads to in the direction away "
+            "from the root). If the length of the edge you are collapsing is "
+            "important, because it came from a tree-making algorithm that sets branch "
+            "lengths according to sequence change, the 'right' thing to do is to add "
+            "the length of the to-be-deleted edge to the lengths of its descendant "
+            "edges when you attach those nodes as children of the node at the root-end "
+            "of the edge you are deleting. If for some reason you also want to discard "
+            "the length of the edge you are collapsing, you can use this "
+            "--discardLengths option. Make sure you really understand the implications "
+            "of doing so, however, because discarding that length will cause the "
+            "patristic distance of these child nodes to be shorter than it was "
+            "estimated to be by the algorithm that made the tree and set the original "
+            "branch lengths. And why would you want to second-guess the algorithm and "
+            "mess with its conclusions?"
+        ),
+    )
+
+    parser.add_argument(
+        "--ladderize",
+        choices=("ascending", "descending"),
+        help="How to (optionally) ladderize the tree after re-rooting.",
+    )
+
+    parser.add_argument(
+        "--format",
+        default="newick",
+        choices=("fasta", "phylip", "newick", "nexml", "nexus"),
+        help="The output format.",
+    )
+
+    parser.add_argument(
+        "--scale",
+        default=1.0,
+        type=float,
+        help=(
+            "Factor to scale the length of the branches leading to the descendants "
+            "of the new root (if --rootNode is used). This can be used to reduce "
+            "the length of very long branches that result from rerooting on a distant "
+            "outgroup. If you use this option for an image in a publication, you of "
+            "course must point out that you have done so!"
+        ),
+    )
+
+    return parser.parse_args()
+
+
+def main():
+    args = parseArgs()
+
+    if not (args.rootNode or args.ladderize or args.collapse):
+        exit(
```

Comment:
This should be `sys.exit`, right?

Reply:
`exit` is a builtin. I learned this from you, I think :-)